raiden/settings.py

```diff
@@ -21,9 +21,7 @@
         "https://raw.githubusercontent.com/raiden-network/raiden-transport"
         "/master/known_servers.main.yaml"
     ),
-    Environment.DEVELOPMENT: (
-        "https://raw.githubusercontent.com/marcosmartinez7/matrix-known-servers/master/known_servers.test.yaml"
-    ),
+    Environment.DEVELOPMENT: ["http://64.225.11.151"],
```

Comment:
why was this value type changed to `list`?

Reply:
Before this change we had a list of yml files to use to get the server urls, now we decided to put the server url directly here instead, the type change is because we don't need to get the yml file and parse it to get the server urls, now we just put the server url's in the list directly and we use them from there.

Reply:
because of the method that parses asumes a yaml, i have created an issue for this comment and the previous one https://jirainfuy.atlassian.net/browse/RLP-962